fix(service-worker): clear service worker cache in safety worker (#43324)
angular/angular: aio/content/guide/service-worker-devops.md

Also included in the `@angular/service-worker` NPM package is a small
script `safety-worker.js`, which when loaded will unregister itself

> from the browser and remove the service worker cache. This script can 

Reviewer: Let's use `caches`:

```suggestion
from the browser and remove the service worker caches. This script can 
```

> This script can be used both to deactivate `@angular/service-worker`
> as well as any other Service Workers which might have been served in
> the past on your site along with the corresponding caches.

Reviewer: Let's move the mention about caches next to `@angular/service-worker` to avoid confusion (since the script only removes ngsw-specific caches - not any caches that might be used by other SW scripts):

```suggestion
This script can be used both to deactivate `@angular/service-worker`
(and remove the corresponding caches) as well as any other Service
Workers which might have been served in the past on your site.
```

Author: @gkalpak I have made the changes and both safety worker and docs updates are not in one commit. Can you please take a look, I hope every thing is fine now. Thank you so much once again for your time and reviewing my PR.
